core/src/i18n/en.yaml

```
@@ -1214,6 +1214,7 @@ service-management:
   title: Service Management
 services:
   cluster-ip: Cluster IP
+  description: <0>Service </0> enables access to an application running on a set of Pods.
```

```suggestion
  description: <0>Service</0> enables access to an Application running on a set of Pods.
```

See https://kyma-project.io/community/guidelines/content-guidelines/08-style-and-terminology/#terminology for Application vs. application.
Also, since the other descriptions containing `<0>` ([example](https://github.com/kyma-project/busola/blob/2a2030193f0c5516512b4d86156b56009bc1d872/core/src/i18n/en.yaml#L28)) don't leave an empty space, I think it's best to delete the extra space.

---

In this case the `application` is right.